custom/wasm.json

```diff
@@ -0,0 +1,49 @@
+{
+  "features": {
+    "BigInt-to-i64-integration": {
+      "wfd-key": "bigInt"
+    },
```

Comment:
Do you think that we should just simplify this to be `"bcd-key": "wfd-key"`?

Reply:
No strong preference. I did it this way so that in the future, we can add more configuration fields, but that might not be necessary.

Reply:
I figured that was the case, but I wasn't sure if you had any configuration options in mind -- I'll leave the format as-is just in case!